src/registrar/assets/src/js/getgov/helpers.js

```diff
@@ -75,3 +75,17 @@ export function debounce(handler, cooldown=600) {
 export function getCsrfToken() {
     return document.querySelector('input[name="csrfmiddlewaretoken"]').value;
 }
+
+/**
+ * Helper function to submit a form
+ * @param {} form_id - the id of the form to be submitted
+ */
+export function submitForm(form_id) {
+    let form = document.getElementById(form_id);
+    if (form) {
+        console.log("submitting form");
+        form.submit();
+    } else {
+        console.error("Form '" + form_id + "' not found.");
```

Comment:
Nitpick: console.warn?

Reply:
I think this one is correctly an error. This would indicate an error when a user attempts and action and it fails because the form is not found in the dom.